teamsyncd/teamsync.cpp

```diff
@@ -212,6 +212,8 @@ void TeamSync::removeLag(const string &lagName)
 
 void TeamSync::cleanTeamSync()
 {
+    SWSS_LOG_NOTICE("Cleaning up kernel LAGs ...");
```

Comment:
Don't you need the `SWSS_LOG_ENTER();` here as well?

Reply:
Thanks, Added.